index.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
 // getting-started.js
+require('dotenv').config();
 
 const mongoose = require('mongoose');
-
-require('dotenv').config();
 const server = require('./src/server');
 
+mongoose.connect(process.env.MONGOOSE_URI, {useNewUrlParser: true, useUnifiedTopology: true});
```

Comment:
I don't know if this is getting messed up because you have this here and then also in your server.js